charts/aspenmesh-collector/Chart.yaml

```diff
@@ -16,3 +16,6 @@ dependencies:
   - name: custom-pod-autoscaler-operator
     version: 1.3.0
     repository: ""
+  - name: metrics-server
+    version: 3.8.3
+    repository: https://kubernetes-sigs.github.io/metrics-server
```

Comment:
I'm not sure whether I did this right; I needed to run `helm dependency update` and it added three tar files below

Reply:
If we are not sure, another way is to run `kubectl apply -f ...` in the one-line script: https://github.com/kubernetes-sigs/metrics-server?

Reply:
I wondered about that, but I thought it'd be simpler to do as a dependency; one less thing in the script, one less error to check for...  not sure